client/src/utils/Store.js

```diff
@@ -99,6 +63,18 @@ export default class Store {
    * }}
    */
 
+  /**
+   *
+   * @param {object} model
+   */
+  flatten = (model) => {
```

Comment:
Might be nice to have a description here.

Reply:
Done.